.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Default Issue template
+about: Create an issue to help us improve
+title: ''
```

Comment:
I propose a placeholder (as I'm going to do on rero-ils): `title: Write a full sentence: Authentication through an external service is not possible.`